.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: node_js
+
+sudo: false
+
+node_js:
+  - "0.10"
+  - "0.12"
+  - "iojs"
+
+script:
+  - npm run lint
+
+matrix:
+  allow_failures:
+    - node_js: "iojs"
```

Comment:
I think our theory on the other fxa-* repos was that iojs is going to be the next Node stable, so it was a way of early detection if the Node+iojs would break our stuff. But yeah, I'm happy to remove the `matrix` and reduce the `node_js` list to whatever targets we want to build against.